### PR TITLE
Support the dbname@hostname style as a database name

### PR DESCRIPTION
**Pull request by hornetmj**
As a virbak request, support the dbname@hostname style for HA environments. Since cubrid-backup-api has not been managed as a jira issue so far, it is modified without registering a jira issue again.

### CODE REVIEW

File: src/include/handle_manager.h [hunk 1]
```diff
@@ -6,7 +6,8 @@
 #include "backup_manager.h"
 
 /* The maximum length of database name is 17 in English. */
```
Comment (joohok) [inline src/include/handle_manager.h:8]:
```suggestion
```

File: src/include/handle_manager.h [hunk 2]
```diff
@@ -6,7 +6,8 @@
...
-#define MAX_DB_NAME_LEN 17
+/* For database name + @ + host name */
+#define MAX_DB_NAME_LEN 512
```
Comment (joohok) [inline src/include/handle_manager.h:10]: How about changing it to 256 ? 
`CUB_MAXHOSTNAMELEN` is already used in cubrid as the maximum length of "dbname@hostname"

**Review by joohok — APPROVED** [review]

File: src/include/handle_manager.h (hunk 2 above)
Comment (hornetmj) [reply src/include/handle_manager.h:10]: Generally, the maximum hostname is 256. CUB_MAXHOSTNAMELEN also just considers the host name. So, I use 512 iso 256.

File: src/include/handle_manager.h (hunk 1 above)
Comment (hornetmj) [reply src/include/handle_manager.h:8]: I leave it as an information.

**Pull request merged**